openhtf/io/output/json_factory.py

```diff
@@ -41,5 +41,6 @@ def convert_to_dict(self, test_record):
         for value in phase['attachments'].itervalues():
           value['data'] = base64.standard_b64encode(value['data'])
     else:
-      as_dict = data.ConvertToBaseTypes(test_record, ignore_keys=('attachments',))
+      as_dict = data.ConvertToBaseTypes(test_record,
```

Comment:
It was [83 characters long](https://google.github.io/styleguide/pyguide.html#Line_length).